language-extensions/python/test/src/CMakeLists.txt

```diff
@@ -33,6 +33,12 @@ add_executable(pythonextension-test
 )
 
 if (${PLATFORM} STREQUAL linux)
+	# Python runtime version used in the linux environment differs from the one used in Windows at this time.
+	# This declaration should no longer be needed once the Windows version is updated to 3.12.
+	#
+	set(PYTHON_VERSION "3.12")
+	set(PYTHON_VERSION_NO_DOT "312")
+
```

Comment:
Once the versions are updated, please update the test containers in our internal tests, and once all pass, release new versions of the RExtension and PythonExtension libraries.

Reply:
Noted. Created work items for these.